### PR TITLE
Allow resources to be updated 

### LINKED ISSUE #664: When a resource is saved a second time it should persist to the same URI
Related to #669

### LINKED ISSUE #669: After first save & publish, subsequent calls should use updateResource
After saving the RDF the first time and the URI is persistented to the Redux state,  any changes in the UI are reflected in the Redux and show up in the **Preview RDF** modal. Clicking the **Save &amp; Publish** button should bypass the select group modal and also do a [`updateResource`](https://github.com/LD4P/sinopia_server/blob/master/sinopia_client/docs/LDPApi.md#updateResource) `sinopia_client` call.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #669
Fixes #664

### CODE REVIEW

File: __tests__/actionCreators.test.js [hunk 1]
```diff
@@ -35,3 +37,9 @@ describe('signedOut', () => {
     expect(signedOut().type).toEqual('SIGN_OUT_SUCCESS')
   })
 })
+
+describe('update', () => {
+  it('returns a function', () => {
+    expect(update()).toBeInstanceOf(Function)
```
Comment (ndushay) [inline __tests__/actionCreators.test.js]: would we not want a test that is a bit more useful than this?  Like, showing what the function does?

File: src/sinopiaServer.js [hunk 2]
```diff
@@ -154,3 +154,11 @@ export const publishRDFResource = async (currentUser, rdf, group) => {
   await authenticate(currentUser)
   return await instance.createResourceWithHttpInfo(group, rdf, { contentType: 'application/n-triples' })
 }
+
+export const updateRDFResource = async (currentUser, uri, rdf) => {
```
Comment (ndushay) [inline src/sinopiaServer.js]: would this new function warrant a test?

File: __tests__/actionCreators.test.js (hunk 1 above)
Comment (jcoyne) [reply __tests__/actionCreators.test.js]: I've added a test, but it's sort of a mess of async mocks.  I'm not sure if it's more helpful.

File: src/sinopiaServer.js (hunk 2 above)
Comment (jcoyne) [reply src/sinopiaServer.js]: This is only making two calls: authenticate and then the server.  I don't know that a test is going to be useful here other than to say: "Here are some mocks that show it's doing exactly what it's doing"

Comment (jcoyne) [reply src/sinopiaServer.js]: Also, this is covered by the integration test.

File: __tests__/actionCreators.test.js (hunk 1 above)
Comment (ndushay) [reply __tests__/actionCreators.test.js]: I see what you mean.

**Pull request merged**